bin/crew

```diff
@@ -757,16 +766,11 @@ def prepare_package (destdir)
 
     # check for conflicts with other installed files
     conflicts = []
-    Dir[CREW_META_PATH + '*.filelist'].each do |packageList|
-      packageName = File.basename packageList, '.filelist'
-      File.readlines(packageList).each do |line|
-        File.readlines('filelist').each do |item|
-          if line == item && packageName != @pkg.name
-            conflicts << [packageName, item.chomp]
-          end
-        end
-      end
-    end
+    @conflictscmd = "grep --exclude #{CREW_META_PATH}#{@pkg.name}.filelist \
```

Comment:
Should we put a message here something like `puts "Checking for conflicts..."`?

Reply:
done. Does that look right?

Reply:
That works!